src/router.js

```diff
@@ -94,37 +143,48 @@ const router = new Router({
   ]
 });
 
-const themes = {
-  'create_org': () => 'secondary',
-  orgs: () => 'secondary',
-  projects: () => 'secondary',
-  'project_create': () => 'secondary',
-  'privacy_policy': () => 'expand',
-  'account': ({ org, project }) => {
-    if(org && project) return 'normal';
-    return 'secondary'
-  },
-}
+const KEYCLOAK_CLIENT_ID = (process.env.KEYCLOAK_CLIENT_ID || 'weni-webapp');
+const KEYCLOAK_REALM = (process.env.KEYCLOAK_REALM || 'weni-staging');
```

Comment:
is it `weni-staging` yet?

Reply:
should not have a default value, because it is mandatory to specify a realm